### PR TITLE
chore(detectors/node): adapted breaking change in @opentelemetry/sdk-node@0.37.0

### PR DESCRIPTION
**Pull request by kirrg001**
Refs https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1459#issuecomment-1498805145

I do no have write permissions for renovate-bot:renovate/otel-core-experimental.
Feel free to close this PR and cherry-pick the commit.

The README files for all detectors needs to be updated too. Let me know if you want me to update them too.

### CODE REVIEW

**Review by Flarna — APPROVED** [review]

**Comment by Flarna** [comment]
An alternative would be that you cherry pick the commit from renovate and add you fix on top of it.
But this variant is also fine. Once merged renovate will rebase on top of your fix and CI is hopefully green then.

File: detectors/node/opentelemetry-resource-detector-instana/README.md
Comment (dyladan) [inline detectors/node/opentelemetry-resource-detector-instana/README.md:48]: I think this is no longer required at all right? The SDK should transparently handle the Async resources

Comment (kirrg001) [reply detectors/node/opentelemetry-resource-detector-instana/README.md:48]: Yeah I wasn't sure earlier. In general it might be good to show both cases? 
#1 Start SDK & load everything in the background and do not care. 
#2 Show how to wait till Otel has fully finished bootstrapping.

File: detectors/node/opentelemetry-resource-detector-instana/README.md
Comment (Flarna) [inline detectors/node/opentelemetry-resource-detector-instana/README.md:67]: `_resource` is private therefore I don't think we should show this be in the readme.

If one has the need to explicit wait for the resource detection some API on `NodeSDK` should be added. But this should be no concern of all the detector readme files and handled outside of this PR.
Alternative is to await the resource detection before SDK creation.

**Comment by Flarna** [comment]
@kirrg001 It seems you didn't allow edits from maintainers on this branch - or it's an org fork where github doesn't allow this (see [here](https://github.com/orgs/community/discussions/5634)).

So seems the burden to merge main is on you.

File: detectors/node/opentelemetry-resource-detector-instana/README.md
Comment (kirrg001) [reply detectors/node/opentelemetry-resource-detector-instana/README.md]: Ok removed this example.

**Comment by kirrg001** [comment]
> or it's an org fork where github doesn't allow this

Yes it is an organization. This is not supported.
I rebased against main. Rdy to merge

**Comment by Flarna** [comment]
At least in the past it was possible to allow it per PR, see [here](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/767#issuecomment-1002119475).

In general I recommend a personal fork but I know that companies often don't allow this.

**Review by pichlermarc — APPROVED** [review]

**Pull request merged**
